pkg/util/clusteroperator/status.go

```diff
@@ -36,6 +37,11 @@ var (
 	statusHandlers = []StatusHandler{}
 )
 
+// ClearHandlers so that test cases don't endlessly add handlers
```

Comment:
This may lead to some flaky tests since tests should be able to be run in parallel.